zig-dxread/src/main.zig

```diff
@@ -0,0 +1,72 @@
+const std = @import("std");
+const fs = std.fs;
+const gpa = std.heap.GeneralPurposeAllocator(.{}){};
+const allocator = gpa.allocator;
+const ArrayList = std.ArrayList;
+const erorset_1 = error{OutOfMemory};
```

Comment:
No need for this error set

Reply:
yeah, this was part of me trying to understand error sets and how to print them. didn't mean to add it here. thanks!